unified_objects.md

```diff
@@ -139,6 +140,4 @@ If indices are needed, use the list above.
 
 ## Gender
 
-A `Gender` is defined as a single-character string. A gender can either be `"m"` or `"f"`. The preferred representation for "no gender" is `null`. If that isn't an option, the preferred alternative is `"-"`.
-
-
+A `Gender` is defined as a lowercased string of either `"m"`, `"f"`, or `"mf"` (denoting the Pokémon can be either gender). It can also be `null` (denoting no gender). If `null` is unsupported, instead use the string `"-"`.
```

Comment:
I don't think that should be part of the spec, as that spec describes what a concrete pokemon instance looks like, not the pokeset format we use. Only our pokeset format has a notion of "possibilities".

Reply:
In that case I misinterpreted the purpose of this specification. Sorry, I'll fix when I get the chance.

Reply:
Using a list of genders per species would probably be more appropriate for this, then